app/src/main/java/com/sun_asterisk/comics_01/screen/read/adapter/ContentAdapter.java

```diff
@@ -0,0 +1,57 @@
+package com.sun_asterisk.comics_01.screen.read.adapter;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import com.bumptech.glide.Glide;
+import com.sun_asterisk.comics_01.R;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ContentAdapter extends RecyclerView.Adapter<ContentAdapter.ContentViewHolder> {
+    private List<String> mLinks;
+
+    public ContentAdapter() {
+        mLinks = new ArrayList<>();
+    }
+
+    public void setData(List<String> links) {
+        mLinks = links;
```

Comment:
check null,
dung add item.
chưa notify change

Reply:
hi. phần notify bỏ bên fetch data and event. để xóa để luôn qua phần bên kia